public-api/src/main/java/org/pmiops/workbench/cdr/model/QuestionConcept.java

```diff
@@ -50,6 +54,16 @@ public static void setGenderStratumNameMap() {
         genderStratumNameMap.put("8570", "Ambiguous");
     }
 
+    public static void setGenderIdentityStratumNameMap() {
+        genderIdentityStratumNameMap.put("1585840", "Woman");
```

Comment:
Are these concept IDs? Where is this mapping originally defined/documneted?

Reply:
i made this mapping myself, looking at the distinct answers to the question and their names in concept table.